src/index.html

```diff
@@ -16,7 +32,370 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
```

Comment:
it should have mix block here

Reply:
Sorry, I didn't understand what needs to be changed.
Can you explain it better?